sourced/ml/cmd_entries/repos2coocc.py

```diff
@@ -3,25 +3,31 @@
 
 from sourced.ml.extractors import IdentifiersBagExtractor
 from sourced.ml.models import OrderedDocumentFrequencies
-from sourced.ml.transformers import Ignition, HeadFiles, UastExtractor, Cacher, UastDeserializer, \
-    CooccConstructor, CooccModelSaver, BagFeatures2DocFreq, Uast2BagFeatures, Counter, \
-    UastRow2Document, LanguageSelector
-from sourced.ml.utils import create_engine
-from sourced.ml.utils.engine import pipeline_graph, pause
+from sourced.ml.transformers import BagFeatures2DocFreq, Cacher, CooccConstructor, \
+    CooccModelSaver, Counter, create_parquet_loader, HeadFiles, Ignition, LanguageSelector, \
+    UastDeserializer, UastExtractor, UastRow2Document, Uast2BagFeatures
+from sourced.ml.utils import create_engine, pause, pipeline_graph
 
 
 @pause
 def repos2coocc_entry(args):
     log = logging.getLogger("repos2coocc")
-    engine = create_engine("repos2coocc-%s" % uuid4(), **args.__dict__)
-    id_extractor = IdentifiersBagExtractor(docfreq_threshold=args.min_docfreq,
-                                           split_stem=args.split_stem)
-
-    ignition = Ignition(engine, explain=args.explain)
-    uast_extractor = ignition \
-        .link(HeadFiles()) \
-        .link(LanguageSelector(languages=args.languages)) \
-        .link(UastExtractor()) \
+    session_name = "repos2coocc-%s" % uuid4()
+    extractors = IdentifiersBagExtractor(docfreq_threshold=args.min_docfreq,
```

Comment:
`extractors` to `extractor`. The only one here. And I prefer old name :)

Reply:
hehe fine will change back :p